fix(hardware-storage-emc-datadomain-snmp): fixed wrong threshold default values in filesystems template CTOR-1343 (#4316)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-emc-datadomain-snmp.md

| Macro                 | Description                                                                                                                                      | Valeur par défaut                                       | Obligatoire |
|:----------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------|:-----------:|

> | TRULYALERT            | Expression to define a truly alert                                                                                                               | %\{severity\} =~ /emergency\|alert\|warning\|critical/i |             |

Author: ```suggestion
| TRULYALERT            | Expression to define an actual alert                                                                                                               | %\{severity\} =~ /emergency\|alert\|warning\|critical/i |             |
```

|:-------------------------|:------------------------------------------------------------------------------------------------------------------------------|
| --filter-counters        |   Only display some counters (regexp can be used). Example to check SSL connections only : --filter-counters='^xxxx\|yyyy$'   |
| --display-alerts         |   Display alerts in verbose output.                                                                                           |

> | --truly-alert            |   Expression to define a truly alert (default: '%\{severity\} =~ /emergency\|alert\|warning\|critical/i').                    |

Author: ```suggestion
| --truly-alert            |   Expression to define an actual alert (default: '%\{severity\} =~ /emergency\|alert\|warning\|critical/i').                    |
```
